src/main/java/org/opentripplanner/apis/gtfs/datafetchers/ItineraryImpl.java

```diff
@@ -100,6 +101,20 @@ public DataFetcher<Double> accessibilityScore() {
     return environment -> NumberMapper.toDouble(getSource(environment).getAccessibilityScore());
   }
 
+  @Override
+  public DataFetcher<Map<String, Double>> emissions() {
+    return environment -> {
+      Emissions emissions = getSource(environment).getEmissions();
+      if (emissions == null) {
+        return null;
+      }
+
+      Map<String, Double> result = new HashMap<>();
+      result.put("co2grams", emissions.getCo2grams());
```

Comment:
@leonardehrenfried Should we create a record for the emissions that we use in the API as this is not really pretty. Although, I'm not sure if  anything throws an exception if we make a naming mistake for a field in the model either.

Reply:
I would prefer a record and a test.

Reply:
I think I got confused here when looking at some other methods. I shall fix it.